### PR TITLE
feat: added service monitor support for graylog chart

### PR DESCRIPTION
**Pull request by ashish1099**
# What this PR does / why we need it

Enable collecting metrics for graylog

- fixes #

# Special notes for your reviewer

# Checklist

- [x] [DCO](https://github.com/KongZ/charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped

### CODE REVIEW

File: charts/graylog/values.yaml [hunk 1]
```diff
@@ -342,6 +342,9 @@ graylog:
     ## If true, prometheus annotations will be attached
     ##
     enabled: false
+    serviceMonitor:
+      enabled: false
+      additionalLabels: {}
```
Comment (KongZ) [inline charts/graylog/values.yaml:347]: Please add missing `interval` and `scrapeTimeout` in value file too. And write some short description of the purpose of them.

**Review by KongZ — COMMENTED** [review]
Please bump chart version and add detail of new settings here

https://github.com/KongZ/charts/blob/main/charts/graylog/README.md#configuration

File: charts/graylog/values.yaml (hunk 1 above)
Comment (ashish1099) [reply charts/graylog/values.yaml:347]: done added now

**Review by KongZ — APPROVED** [review]

**Comment by ashish1099** [comment]
Have signed my commits :)

**Pull request merged**